mne_bids/write.py

```diff
@@ -46,6 +46,16 @@ def _is_numeric(n):
     return isinstance(n, (np.integer, np.floating, int, float))
 
 
+def _stamp_to_dt(utc_stamp):
+    """Convert timestamp to datetime object in Windows-friendly way."""
+    # The min on windows is 86400
```

Comment:
min of what is 86400? I know I linked to the issue but now I forgot -- just trying to think what might be a friendly way to remind a future developer what this is all about.

Reply:
I don't know, I didn't write that, just copied it over

Reply:
I think that's related to the page that you sent me earlier actually, in there they said Windows throws an error for values less than 86400 because that's before Jan 1, 1970.

Reply:
copied from where?

Reply:
I'd suggest adding a link to the issue if it's not easy to explain. See how we have done it elsewhere in the repository